projects/python-client/src/datapane/client/api/user.py

```diff
@@ -153,14 +131,20 @@ def hello_world():
 
 def token_connect(open_url: str, action: str, server: str) -> t.Optional[str]:
     """
-    Create a signup token, and prompt the user to login/signup while polling for completion.
-    Then log the user into the CLI with the retrieved API token
+    Creates a login token, and prompt the user to login while polling for completion.
+    Then log the user into the CLI with the retrieved API token.
+
+    NOTE: throughout this function we refer to signup tokens however this function will
```

Comment:
is this `NOTE` still valid?

Reply:
No, it is no longer valid. I managed to refactor the BE so there are no mentions of sign up tokens